### ISSUE TITLE
[IGNORE, DUPLICATE OF #54]

### ISSUE THREAD

**Issue by vikasithsai** [cse442-at-ub/project_s23-cinco#55]
Acceptance Test
Test 1:

Then click on the option "login " to view your account
or else click on the option "logout" to exit.

**Comment by junqiwu02** [comment]
Tasks should be a blocking dependency (not blocked by)

**Comment by junqiwu02** [comment]
Add this as a blocking dependency of a user story. The test also needs to show how to open that page.
